ramp.yml

```diff
@@ -5,7 +5,7 @@ description: Serving tensors and executing deep learning graphs
 homepage: https://oss.redislabs.com/redisai/
 license: GNU Affero General Public License v3.0
 command_line_args: ""
-min_redis_version: "5.0"
+min_redis_version: "5.0.7"
 min_redis_pack_version: "5.4"
```

Comment:
@rafie @gkorland should this not be 5.4.11 ?

Reply:
the pack version

Reply:
You tell me :-)

By the way, I see the license is also outdated. I can update both, we go with 5.4.11 then?